eq-author-api/middleware/loadQuestionnaire.js

```diff
@@ -9,7 +10,11 @@ module.exports = async (req, res, next) => {
 
   const questionnaire = await getQuestionnaire(questionnaireId);
 
-  if (questionnaire && questionnaire.isPublic === false) {
+  const notAdmin = !get(req, "user.admin", false);
```

Comment:
Instead of using inverted logic could we do something like:
```
const isAdmin = get(req, "user.admin")

if(isAdmin){
	req.questionnaire = questionnaire;
	next();
	return; 
}
```
and just short circuit the whole thing?